src/query/term_query/term_weight.rs

```diff
@@ -124,7 +131,7 @@ impl TermWeight {
         };
```

Comment:
i'm still not sure how the JSON path is being used parsed through for TermQuery, doesn't seem very clear, but it gets the inverted index in the same way.